platform/core/src/utils/measurementFilters.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Returns a filter function which filters for measurements belonging to both
+ * the study and series.
+ */
+export function filterTracked(trackedStudy: string, trackedSeries) {
+  return measurement => {
```

Comment:
does it matter tracked? can't it be `filterByStudyAndSeries` and still does the job?